src/main/java/de/tum/in/www1/artemis/domain/Bonus.java

```diff
@@ -0,0 +1,83 @@
+package de.tum.in.www1.artemis.domain;
+
+import javax.persistence.*;
+
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * A bonus source for an exam that maps bonus from another course or exam to the target exam
+ */
+@Entity
+@Table(name = "bonus")
+@Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class Bonus extends DomainObject {
+
+    /**
+     * Can be either +1 or -1 to add or subtract bonus.
+     */
+    @Column(name = "weight")
+    private double weight;
```

Comment:
💡Idea: Add validation to make sure this range is always kept

Reply:
`Math.signum` in the [setter](https://github.com/ls1intum/Artemis/pull/5222/files/fff321d053106eed522c226ca4d58939657bb162#diff-5b157b83b9950d2803e45d4f4be1d2dd2321ef1a87261baab34855802cb145adR72-R74) takes care of it.